docs/source/blocks/settings.md

```diff
@@ -9,12 +9,12 @@ myst:
 
 # Blocks settings
 
-You should make Volto aware of your custom blocks.
-Since Volto have its own set of default blocks, you should extend them by adding your custom ones in your project configuration object.
+Volto has a set of default blocks.
+You can extend it by adding your custom blocks in your project's configuration object.
 
 ## Configuring a new block
 
-So we add these lines to the `src/config.js`:
+To extend the default set of blocks, you add the following lines to the `src/config.js`:
```

Comment:
Not anymore, I'd say we point people to use the config in add-on instead of the old project approach.